monero-outreach-docs/translations/fr/merchants/merchant_FAQ_fr.md

```diff
@@ -0,0 +1,57 @@
+- **Aucune Transaction Retardée** - Monero fait appel à une taille de bloc dynamique, tandis que la taille de bloc de Bitcoin est fixée et statique. Si le réseau Bitcoin est très sollicité et les blocs sont remplis, ceux qui désirent y inscrire une transactions doivent payer des frais supplémentaires ou voir la transaction reléguée à un prochain bloc.
+- **Sans Risque** - Puisque Monero est privé en tout temps, et ainsi pleinement fongible, personne ne peut discriminer contre l'historique d'un XMR particulier. Le refus d'acceptation d'un Bitcoin à l'historique "teintée" est d'ailleurs un problème croissant pour les marchands qui acceptent le Bitcoin.
+
+### _Comment les frais de Monero se comparent-ils aux frais des cartes de débit/crédit?_
+
+Avec Monero, c'est l'émetteur qui paie les frais de transaction. Ce sera toujours totalement gratuit d'accepter des XMR. Présentement, il coûterait à peu près US$ 0.02 à vos clients pour payer avec Monero. Cette méthode de paiement est ainsi beaucoup moins coûteuse que le paiement par carte de crédit, en plus d'être finalisée plus rapidement.
+
+| Frais | American Express | Mastercard | Visa | Débit |
+|--|--|--|--|--|
+| Frais de transaction | 2.5% - 3.5% | 1.55% - 2.6% | 1.43% - 2.4% | 0.30% - 2.75% |
+| Frais d'interchange | -- | 0.08% | 0.09% | 0.05% + $0.21 0.70% + $0.15 |
+| Frais de réseau | 0.15% | 0.08% | 0.09% | 0.75% - 0.90% |
+| Frais "Switch" | -- | -- | -- | $0.025 - $0.075 |
+| Frais pour achat de $10 | **$0.27 - $0.37** | **$0.18 - $0.28** | **$0.16 - $0.26** | **$0.35 - $0.66** |
+| Frais pour achat de $100 | **$2.65 - $6.65** | **$1.78 - $2.83** | **$1.61 - $2.58** | **$3.45 - $6.60** |
+| Frais pour achat de $1,000 | **$26.50 - $66.50** | **$17.80 - $28.30** | **$16.10 - $25.80** | **$34.50 - $66.00** |
+
+Les frais de transaction associés aux paiement par débit/crédit varieront selon la nature de votre entreprise et selon votre fournisseur de services de paiement, mais les transactions en utilisant Monero engendreront des frais de transactions allant de 7x à 3023x moins chers qu'une transaction par débit/crédit.
+
+Vous pouvez être appelés à payer des frais additionnels si le numéro de carte est entré manuellement, ou si la carte de débit est glissée dans le terminal, ou s'il s'agit d'une transaction internationale. Il est également courant que le fournisseur de services de paiement vous chargent des frais mensuels ou annuels.
+
+### _Les transactions avec Monero sont-elles moins dispendieuses qu'avec Bitcoin?_
+
+Aussi complexe que les transactions par crédit/débit peuvent être, les frais tendent à être réguliers. Le Frais d'Interchange est régi par la loi et mis à jour deux fois par année. En comparaison, les frais d'utilisation de Bitcoin sont erratiques, irréguliers, et augmentent lorsque le réseau est sollicité. Les frais de transaction pour Monero, quant à eux, sont demeurés bas et stables depuis l'introduction de Bulletproofs en 2018.
```

Comment:
1.
deux fois par année.   >>> deux fois par an   

Reason: Sound better to me (but could be due to a regional difference between the translator and the reviewer)

2.
Original sentence
les frais d'utilisation de Bitcoin sont erratiques, irréguliers, et augmentent lorsque le réseau est sollicité  

Suggested sentence:
les frais d'utilisation de Bitcoin ne sont en aucun cas réguliers, et augmentent lorsque le réseau est sollicité    

Reason: Translation choice which I think is closer to the original meaning.

Reply:
Integrated both recommended changes.